app/src/main/res/layout/fragment_browser.xml

```diff
@@ -47,9 +47,6 @@
         android:id="@+id/cursorView"
         android:layout_width="@dimen/remote_cursor_size"
         android:layout_height="@dimen/remote_cursor_size"
-        android:layout_gravity="top|start"
-        android:src="@drawable/cursor_full"
-        android:scaleType="centerInside"
```

Comment:
nit: I can't find out what the default scaleType is: we should leave this or set it in code.